bot/extensions/quotes.py

```diff
@@ -0,0 +1,22 @@
+from discord.ext.commands import Cog, command
+from discord import Embed
+from json import loads
+from requests import get
+
+class Quote(Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @command(name='quote', help='sends a inspirational quote', usage='quote')
+    async def quote_command(self, ctx):
+        response = get('http://api.forismatic.com/api/1.0/?method=getQuote&format=json&lang=en')
```

Comment:
Maybe this should go in a CONFIG file once we get the cleverer config system up and running.